app/scripts/controllers/transactions.js

```diff
@@ -577,6 +594,13 @@ angular.module('openhimConsoleApp')
 
       // execute refresh if no errors
       if($scope.ngError.hasErrors === false) {
+        
+        // Set dateApplied flag if a date filter is being applied
+        if($scope.settings.filter.startDate || $scope.settings.filter.endDate) {
+          $scope.dateApplied = true;
+        } else {
+          $scope.dateApplied = false;
+        }
```

Comment:
The date filter is only triggered on blur, but the scope variable is set on date selected, resulting in the (Filters Applied) notification being displayed before the filter is applied.